microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md

```diff
@@ -120,12 +120,12 @@ Microsoft Managed Desktop devices must be allowed to enroll in Intune.
 
 **Not ready**
 
-Follow the steps in [Set enrollment restrictions](https://docs.microsoft.com/mem/intune/enrollment/enrollment-restrictions-set) to change the setting to **Allow**.
+You currently have at least one enrollment restrictions policy configured to prevent Windows devices from enrollment into Intune. Follow the steps in [Set enrollment restrictions](https://docs.microsoft.com/mem/intune/enrollment/enrollment-restrictions-set) for each enrollment restriction policy that targets Microsoft Managed Desktop users and change the Windows (MDM) setting to **Allow**.  Note that personally owned WIndows (MDM) devices can be set to **Block**. 
```

Comment:
```suggestion
You currently have at least one enrollment restrictions policy configured to prevent Windows devices from enrollment into Intune. Follow the steps in [Set enrollment restrictions](https://docs.microsoft.com/mem/intune/enrollment/enrollment-restrictions-set) for each enrollment restriction policy that targets Microsoft Managed Desktop users and change the Windows (MDM) setting to **Allow**.  Note that personally owned Windows (MDM) devices can be set to **Block**. 
```